src/main/java/uk/gov/hmcts/reform/sscs/helper/processing/ProcessHmcMessageHelper.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.sscs.helper.processing;
+
+import uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingGetResponse;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.AWAITING_LISTING;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.CANCELLED;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.EXCEPTION;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.LISTED;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.UPDATE_SUBMITTED;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.ListingStatus.CNCL;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.ListingStatus.FIXED;
+
+public final class ProcessHmcMessageHelper {
+
+    private ProcessHmcMessageHelper() {
+    }
+
+    public static boolean stateNotHandled(HmcStatus hmcStatus, HearingGetResponse hearingResponse) {
```

Comment:
Id really like to see us use a component rather than static methods in a class
Can e change this please?

Reply:
Its helper class and it just checking the static values only